docs/src/datamanagment/metadata/metadata-search.md

```diff
@@ -0,0 +1,324 @@
+
+!!! info
+    Direct queries by tag name (e.g., repo-metadata.v1.2.system.object_metadata) are not currently supported — use the 
+    commit ID instead.
+
+### Example Queries
+
+This section showcases how to use lakeFS Metadata Search to answer different types of questions using standard SQL.
+
+For simplicity and readability, the examples are written in Trino SQL. If you're using another engine (e.g., DuckDB, 
+Spark, or PyIceberg), you may need to adjust the syntax accordingly.
+
+#### Object Annotation and Labeling
+
+##### Filter by Object Labels
+
+The following example returns images labeled as dogs that are not in a sitting position:
+```sql
+USE "repo-metadata.main.system";
+SHOW TABLES;
+
+SELECT * FROM object_metadata
+WHERE path LIKE `%.jpg`
+  AND user_metadata['animal'] = 'dog'
+  AND user_metadata['position'] != 'sitting';
+```
+
+##### Filter by Object Labels on past commit 
+
+This example demonstrates how to write reproducible queries by referencing a specific lakeFS commit.
+It returns images labeled as dogs that are not sitting, based on the metadata state at that commit:
+
+```sql
+USE "repo-metadata.commit-dc3117ec3a727104226c896bf7ab9350ee5da06ae052406262840e9a4a8c9ffb.system";
+SHOW TABLES;
+
+SELECT * FROM object_metadata
+WHERE path LIKE `%.jpg`
+  AND user_metadata['animal'] = 'dog'
+  AND user_metadata['position'] != 'sitting';
+```
+
+##### View Metadata from AI-Powered Annotators
+
+Assume that any object annotated by an AI-powered tool includes the object metadata key-value pair `source: autolabel` to 
+indicate its origin. The following example returns all such AI-annotated objects: 
+
+```sql
+USE "repo-metadata.main.system";
+
+SELECT *
+FROM object_metadata
+WHERE user_metadata['source'] = 'autolabel';
+```
+
+#### File Properties and Storage
+
+##### Filter by File Extension & Size
+
+Find all `.png` files larger than 2MB.
+
+```sql
+USE "repo-metadata.main.system";
+
+SELECT *
+FROM object_metadata
+WHERE path LIKE '%.png'
+  AND size::INT > 2000000;
+```
+
+#### Filter objects by addition Time
+
+This example finds all objects added in the last 7 days.
+
+```sql
+USE "repo-metadata.main.system";
+
+SELECT *
+FROM object_metadata
+WHERE creation_date >= current_timestamp - interval '7' day;
+```
+
+#### Audit and Governance
+
+##### Detect Errors in Sensitive Data Tagging
+
+Assume all objects under `customers/` must have user metadata `PII=true`. 
+This example returns objects where `PII=false`, or PII key is missing. 
+
+```sql
+USE "repo-metadata.main.system";
+
+SELECT *
+FROM object_metadata
+WHERE path LIKE 'customers/%'
+  AND (
+    user_metadata['PII'] = 'false'
+        OR user_metadata['PII'] IS NULL
+    );
+```
+
+## Limitations
+
+* Applies only to objects added or modified after the feature was enabled. Existing objects before that point are not indexed. 
+* No direct commit and tag support: To query by commit or tag, see the [writing reproducible queries](#writing-reproducible-queries)
```

Comment:
What do you mean by "direct commit"? earlier you showed an example of querying by commit ID, no?

Reply:
to query by commit id you need to prefix it with a `commit-` prefix. I changed this and will try to add clarity to the reproducible queries section